docs/src/manual/constraints.md

```diff
@@ -1238,8 +1238,9 @@ julia> @constraint(model, x in MOI.ExponentialCone())
 ## Set inequality syntax
 
 For modeling convenience, the syntax `@constraint(model, x >= y, Set())` is
```

Comment:
Note that as per https://docs.julialang.org/en/v1/manual/unicode-input/
`⪰` can be "trivially" inputted as `\succeq`.

Wouldn't it be possible to:
```
@constraint(model, x \succeq y)
# error: specify one of the conic sets
```
```
@constraint(model, x >= y, <... Some conic set ...>)
# warning: use \succeq instead
```
```
@constraint(model, x \succeq y, <... Some non-conic set ...>)
# warning: use >= instead
```
```
@constraint(model, x >= y)
# becomes @constraint(model, x - y, Nonnegatives)
```
```
@constraint(model, x <= y)
# becomes @constraint(model, x - y, Nonpositives)
```
Is the concern that someone writes `@constraint(model, x >= y)` thinking it means `PSDCone`,
it silently works, and doesn't do what's expected? Or is there some other concern?

Reply:
> Is the concern that someone writes @constraint(model, x >= y) thinking it means PSDCone,
it silently works, and doesn't do what's expected? 

Yes.

We won't be adding support for `\succeq`. There is too much risk of confusion.

Reply:
:/ I really feel such a decision should be made with a larger headcount than i see here.

Reply:
I'll open a new issue